stop.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 echo "Shutting down Akeneo service"
-source .env
-export SOURCE_PATH="${WORKING_DIR}"
 docker-compose down
```

Comment:
The original goal was to include `/etc/hosts` modifying scripts and possible post run cleanup actions, but this has been scaled back for now. And honestly if some one care enough about a script, they might at least glance through to see if it's risky or good for them.